components/brave_wallet/browser/internal/hd_key_zip32_unittest.cc

```diff
@@ -76,4 +76,55 @@ TEST(HDKeyZip32Test, GenerateFromSeed) {
   }
 }
 
+TEST(HDKeyZip32Test, FullViewKey) {
```

Comment:
Sorry, didn't understand you, in this test we don't have "accounts" we have different derivation pathes and every derived child has fvk

Reply:
Yeah, sorry was using the term they use in the test vectors: https://github.com/zcash/zcash-test-vectors/blob/db9c9b9519a32859a46bbbc60368e8741fe629c4/test-vectors/rust/unified_full_viewing_keys.rs#L52-L68

Just noticed that this one was dropped and figured it might be useful to have a test vector where the `orchard_fvk_bytes` is `none`.

Reply:
So more specifically, what I'm curious to know is can we add tests for derived path 2 and 3 to make sure we also get none?

Reply:
Tests in the zcash-test-vectors differs from what we are testing.
They construct "account" from different parts.
For ex: transaprent + orchard, or tr + or + sapling, or tr + sapling.
If account doesn't have orchard part, orchard_fvk will be empty.
In our case hd_key_zip32 always has orchard_part so it always should have fvk